website/src/site/markdown/publications.md

```diff
@@ -0,0 +1,39 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Publications
+
+The main paper on REEF is:
+
+**REEF: Retainable Evaluator Execution Framework**, Markus Weimer et al. *ACM SIGMOD Industrial Track, 2015*. [`[pdf]`](papers/2015-SIGMOD.pdf)[`[bib]`](bib/reef.bib)
+
+
+Other publications, with most recent at the top:
```

Comment:
Do we need to say "with most recent at the top" if each paper has a year associated with it? The phrase looks a bit weird to me